src/kubeseal_auto/host.py

```
@@ -1,67 +1,238 @@
+
+    Args:
+        version: The version string (e.g., 'v0.26.0' or '0.26.0').
+
+    Returns:
+        The version string without leading 'v' (e.g., '0.26.0').
+
+    Raises:
+        ValueError: If version is None, empty, or doesn't match semantic versioning.
+    """
+    if not version:
+        raise ValueError("Version string cannot be None or empty")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)